src/utils/helper-functions.tsx

```diff
@@ -288,3 +288,6 @@ export const getGroupedByDate = <T extends { createdAt: string }>(
 
     return result
   }, [])
+
+export const getAttachmentExtension = (fileName: string): string =>
+  fileName.split('.').reverse()[0]
```

Comment:
I suggest rewriting this function to split filename into `basename` and `extension`. For instance filename `test.file-bla-bla-bla.pdf` has to be splitted into basename `test.file-bla-bla-bla` and extension `pdf`. Then we can reuse it in `EditAttachmentModal.constants.ts`

Reply:
rewrote this function according to your notes, thank you!

Reply:
Became much better!